test_psa_target.py

```diff
@@ -348,7 +361,14 @@ def _get_parser():
     )
 
     parser.add_argument(
-        "--no_sync",
+        "-r",
+        "--run",
+        help="Run on the target only",
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "--no-sync",
```

Comment:
As this is changed to `--no-sync` - this needs to be changed in other places

Reply:
Apparently have an issue with build --no-sync binaries

Reply:
No need - Python's argument parser automatically converts the `-` to `_` in the parsed variable name, so you get `args.no_sync`. It's because Python doesn't allow `-` in variable names.
I updated README already.

Reply:
Never mind, there is a problem with my setup